advanced_html_css/accessibility/semantic_html.md

```diff
@@ -76,6 +76,10 @@ Headings are the `<h1>` through `<h6>` elements, and like the name implies, thes
 - `<nav>`
 - `<section>`
 
+As an example, consider this image which shows how this lesson in particular uses semantic HTML for the different areas in the page. If you want to examine the page in more detail, feel free to use the developer tools to do so.
+
+<img src='https://github.com/TheOdinProject/curriculum/assets/154856872/232855c8-13cd-4119-a1f3-f1ebe05fd40c' alt='example of how the typical odin lessons html is structured'>
```

Comment:
This needs to be updated to Markdown img syntax.

For the alt text, maybe we can tweak that to, "How a typical Odin lesson uses semantic HTML by including the nav, header, main, section, aside, and footer elements"

Lastly, @wise-king-sullyman are we still using the method from our wiki on [adding images](https://github.com/TheOdinProject/curriculum/wiki/Adding-Images-to-the-Curriculum)? If so, we should probably consider another means to avoid needing to open 2 PRs just for adding images.

Reply:
@nikitarevenco I think that looks good

Reply:
> @nikitarevenco I think that looks good

Alright, I've added this image

Reply:
@thatblindgeye AFAIK yeah, using statically as outlined in the docs is still the best approach we have. I don't agree that something better would be..... better though.

Reply:
@nikitarevenco let's follow the instructions on that wiki link I posted a few comments above. Basically you'll just want to:

- Create a couple of new directories so that end up with: `advanced_html_css/accessibility/semantic_html/imgs`
- Add the image file (`.png` preferred) to that new imgs directory
- Update the img url in the lesson to the image path

Then we'd need a followup PR to add the statically links

Reply:
> @nikitarevenco let's follow the instructions on that wiki link I posted a few comments above. Basically you'll just want to:
> 
> * Create a couple of new directories so that end up with: `advanced_html_css/accessibility/semantic_html/imgs`
> * Add the image file (`.png` preferred) to that new imgs directory
> * Update the img url in the lesson to the image path
> 
> Then we'd need a followup PR to add the statically links

[Made a PR that adds the images to the repo](https://github.com/TheOdinProject/curriculum/pull/27488)